service/idea_service.py

```diff
@@ -0,0 +1,26 @@
+from repository.orm.Idea import Idea
+
+
+class IdeaService:
+    def __init__(self):
+        return
+
+    def get_all(self):
```

Comment:
@adamaslan I would recommend you take a look at paginate so it does not return  1000s of records you can set a default.

https://flask-sqlalchemy.palletsprojects.com/en/2.x/api/

Reply:
Oh very cool...looking to use Axios on the frontend - working on this with Trisha who ordered massive rugs from Wayfarer